core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1518,12 +1520,12 @@ private void parseSelectClause(GenericLexer lexer, QueryModel model) throws SqlE
 
             if (tok != null && Chars.equals(tok, ';')) {
                 alias = createColumnAlias(expr, model);
-                //tok = optTok(lexer);
             } else if (tok != null && columnAliasStop.excludes(tok)) {
                 assertNotDot(lexer, tok);
 
```

Comment:
To create alias one doesn't need to use "as" keyword , it's optional . 
These sqls are equivalent : 
```sql
select ts "tt" from X 
select ts tt from X ; 
```